UnityProject/Storybook/Assets/Code/Scripts/Game/World/MapManager.cs

```diff
@@ -872,4 +894,50 @@ private void _resetRoomDataAStar()
             }
         }
     }
+
+    public RoomObject PlaceStartRoom()
+    {
+        Location start = new Location(m_startPoint.x, m_startPoint.y);
+        return PlaceRoom(start);
+    }
+
+    private void _sendMap()
```

Comment:
This should be handled through a SyncProperty. You can make a SyncProperty a PhotonStream now which will allow you to pack larger amounts of data to send.